Strategies/ITVision.md

```diff
@@ -0,0 +1,168 @@
+Documents are rare, records management systems are obsolete.
+Efficient digital processes have rendered paper records obsolete. This means:
+
+- Briefing notes are digitized
+- Corporate services, such as HR and Finance, do not require ESDC staff to provide information on their first day of work as the hiring process already have that information
+
+Instead the information used by ESDC's staff is presented online via a User Experience that seamlessly integrates the combination of data from multiple sources.
+
+ESDC, with partnership with the union, has changed IT talent management for the federal government to improve recruitment, retention of talent, and wellbeing by:
+
+- allowing experienced and valuable CS personnel to reach a salary bracket reflective of their worth, up to CS-05 level, without requiring to move into management or advisory role (e.g. expert coder, data scientist)
+- promoting *easy* lateral moves across streams in order to be exposed to new challenges and perspectives
+- allocating 20% of time to continuous improvement, as opposed to yearly class-based training
+- promoting creativity over status quo thinking
+- Cyber security is built in ESDC's continuous improvement, onboarding training, and performance agreements to assure the public that ESDC's staff can be trusted with their information.
+
+ESDC's IT community's talent revolves around a continuous improvement culture, fostering and supporting an environment for staff to learn new technologies and how they may be applied to their daily work.
+
+ESDC's IT community is diverse to reflect the full breadth of the talent pool.
+This is achieved by proactively promoting technology opportunities to under-represented groups.
+
+## 6. Good digital governance
+
+ESDC recognizes upcoming digital leaders by identifying employees that demonstrates leadership qualities, embrace the digital mindsets, can adapt the ever more rapid pace of change, show empathy to end users, and are able to keep pace with technology lingo and concepts.
+
+ESDC is using product management to ensure its services and associated IT solutions are able to manage technical debt, security patching, bug fixes, and feature enhancements rapidly.
+
+ESDC is using agile project management to coordinate large changes across the enterprise where investments are aligned strategically with the direction of the GC-wide enterprise architecture.
+Depending on their complexity, Agile projects makes use of a concept phase prior to defining the solution.
+
+Enterprise Architecture and peer reviews (e.g. Enterprise Architecture Review Boards) are done by specialists in their fields, that can provide advice on technology choices, data management, and cyber security investments decisions.
+
+ESDC is measured against the Digital Performance Measurement Framework and Maturity Model and is continuously improving its maturity level for each of the maturity domains:
```

Comment:
Add link to "Digital Performance Measurement Framework and Maturity Model" (if existing)?

Reply:
Framework isn't yet live but will be shortly I think.